pkg/webhooks/server.go

```diff
@@ -237,30 +239,35 @@ func (ws *WebhookServer) handleMutateAdmissionRequest(request *v1beta1.Admission
 			},
 		}
 	}
-
-	// MUTATION
-	// mutation failure should not block the resource creation
-	// any mutation failure is reported as the violation
-	patches := ws.HandleMutation(request, resource, policies, roles, clusterRoles)
-
-	// patch the resource with patches before handling validation rules
-	patchedResource := processResourceWithPatches(patches, request.Object.Raw, logger)
-
-	if ws.resourceWebhookWatcher != nil && ws.resourceWebhookWatcher.RunValidationInMutatingWebhook == "true" {
-		// VALIDATION
-		ok, msg := ws.HandleValidation(request, policies, patchedResource, roles, clusterRoles)
-		if !ok {
-			logger.Info("admission request denied")
-			return &v1beta1.AdmissionResponse{
-				Allowed: false,
-				Result: &metav1.Status{
-					Status:  "Failure",
-					Message: msg,
-				},
+	var patchedResource []byte
```

Comment:
I don't see where the value is assigned?

Reply:
I think it's assigned if the version is satisfied

Reply:
What if version < 1.14.0 and there's a generate policy? What been passed into `HandleGenerate`?